tests/sasl_plain_ut/sasl_plain_ut.c

```diff
@@ -186,32 +214,30 @@ TEST_FUNCTION(saslplain_create_with_NULL_authzis_succeeds)
 TEST_FUNCTION(saslplain_destroy_frees_the_allocated_memory)
 {
 	// arrange
-	amqp_frame_codec_mocks mocks;
 	SASL_PLAIN_CONFIG sasl_plain_config = { "test_authcid", "test_pwd", NULL };
 	CONCRETE_SASL_MECHANISM_HANDLE sasl_plain = saslplain_create(&sasl_plain_config);
-	mocks.ResetAllCalls();
+	umock_c_reset_all_calls();
 
-	EXPECTED_CALL(mocks, amqpalloc_free(IGNORED_PTR_ARG));
-	EXPECTED_CALL(mocks, amqpalloc_free(IGNORED_PTR_ARG));
+	EXPECTED_CALL(gballoc_free(IGNORED_PTR_ARG));
+	EXPECTED_CALL(gballoc_free(IGNORED_PTR_ARG));
 
 	// act
 	saslplain_destroy(sasl_plain);
 
 	// assert
-	// uMock checks the calls
+    ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
 }
 
 /* Tests_SRS_SASL_PLAIN_01_006: [If the argument concrete_sasl_mechanism is NULL, saslplain_destroy shall do nothing.] */
 TEST_FUNCTION(saslplain_destroy_with_NULL_handle_does_nothing)
 {
 	// arrange
```

Comment:
Not needed as it is done before each test function in the test function init.

---
In reply to: [104449908](https://github.com/Azure/azure-uamqp-c/pull/35#discussion_r104449908) [](ancestors = 104449908)